docs/SSR.md

```diff
@@ -0,0 +1,31 @@
+# Server Side Rendering (SSR)
+
+[<< Back to index](./index.md)
+
+The FSXA-PWA project is based on the [nuxt.js](https://nuxtjs.org) framework which supports server side rendering ([SSR](https://ssr.vuejs.org/#what-is-server-side-rendering-ssr)). This means that as opposed to the single page application, the server does not just send json data but instead a fully rendered page to the browser. This allows for search engines to crawl the pages easily. Most of the javascript code will be executed on the server which means that the time until content appears on the screen is much faster. On the other hand switching from one page to another takes longer than in a single page application.
```

Comment:
The rendering of a new view or page *does* take longer when using SSR than when rendered on the client because SPAs usually update only some parts of the page instead of all of the page. See also [here](https://medium.com/@rohit_/ssr-vs-csr-the-ever-on-going-debate-e124021bbdb6#1ea6) or [here](https://blog.logrocket.com/server-side-rendering-with-vue-and-nuxt-js-64a6d99f5a82#f0da)

However one can argue, that the issue might be too complex to be mentioned here. The json data was actually more directed towards caas or other data sources, not the webserver serving the page. The caas json data goes directly to the SPA which then generates the html around it. As opposed to SSR where the json data is being requested on the server where the javascript code then generates html around it. I will try to clarify this issue because I think this is vital to understanding the differences between SSR and CSR

Reply:
I guess we have different sources. 
In this [article](https://vueschool.io/articles/vuejs-tutorials/nuxt-ssr-optimizing-tips/) the author does have the following opinion: 
> It’s important to mention that the server-side rendering process happens only when we directly enter the website (or refresh it). Once hydrated, the app behaves like a normal client-side rendered SPA.

And that coincides with a video that I've watched some time ago. 

Now i understand your sentence about the json data but like you said yourself it will need some clarification.

Reply:
Yes I added this passage to the text now and described it as a hybrid approach.